go/kbfs/kbfsgit/runner.go

```diff
@@ -1970,6 +1971,34 @@ func (r *runner) handleLFSUpload(
 	return r.waitForJournal(ctx)
 }
 
+func (r *runner) handleLFSDownload(
+	ctx context.Context, oid string) (localPath string, err error) {
+	fs, err := r.makeFS(ctx)
+	if err != nil {
+		return "", err
+	}
+	err = fs.MkdirAll(lfsSubdir, 0600)
+	if err != nil {
+		return "", err
+	}
+	fs, err = fs.ChrootAsLibFS(lfsSubdir)
+	if err != nil {
+		return "", err
+	}
+
+	// Put the file in a temporary location; lfs will move it to the
```

Comment:
Is this what other LFS implementations do? seems a little clunky. Not that I have a better idea, but e.g. could weird permissions or disk space limitations make this fail?

Reply:
Yeah this is actually how it's defined in the spec.  But note that LFS does a strict move, not a copy-then-delete, so you have to download the file onto the same partition as the git repo so it should all have the same permissions and disk space limitations as the download to the temp file.